cairosvg/helpers.py

```diff
@@ -383,7 +383,10 @@ def size(surface, string, reference='xy'):
 
     for unit, coefficient in UNITS.items():
         if string.endswith(unit):
-            number = float(string[:-len(unit)])
+            try:
+                number = float(string[:-len(unit)])
+            except (ValueError, TypeError, IndexError) as e:
+                number = 0.0
```

Comment:
It would be good to add a warning here (don't forget to `import warning`):
`warnings.warn("Numeric value is invalid. Value is set to 0")`

Reply:
Thanks, I`ve added it